_posts/2021-05-05-PC-Engines-Firmware-v4-13-0-6.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: post
+title:  "PC Engines apu coreboot Open Source Firmware v4.13.0.6"
+date:   2021-05-05
+categories: Firmware
+---
+
+# PC Engines apu coreboot Open Source Firmware v4.13.0.6
+
+## Key changes
+
+Mainline:
+
+1. **Rebased** with official coreboot repository **commit** a4c09c5.
+
+## Statistics
+
+![Files Changed](https://cloud.3mdeb.com/index.php/s/4mrBz3eRSyDyrms/preview)
+
+The chart shows the total files changed from release tag against the rebase
+point of given release specified in CHANGELOG (CHANGELOG.md and gitlab-ci.yml
+excluded from statistics). Check the statistics with:
+
+```
+git diff --stat a4c09c5 ':(exclude).gitlab-ci.yml' ':(exclude)CHANGELOG.md' ':(exclude).gitlab-ci/regression.sh'
+```
+
+`106 files changed, 4422 insertions(+), 404 deletions(-)`
+
+![Process of mainlining](https://cloud.3mdeb.com/index.php/s/3eRBCMfW2jjdWxs/preview)
+
+The chart represents the total line added and deleted on the PC Engines
+coreboot fork against the rebase point for a given release.
+
+## Testing
+
+* [PC Engines hardware configuration matrix](https://cloud.3mdeb.com/index.php/s/oWDDFxNDZBtJDQP/preview) - hardware configurations available for testing in 3mdeb laboratory.
+
+* [PC Engines release validation results](https://3mdeb.us16.list-manage.com/track/click?u=fce95b885fc13fbf1db611816&id=96d9b426c0&e=16ffa34a09) - please note there are separate sheets for each board-release.
+
+![Mainline test results](https://cloud.3mdeb.com/index.php/s/qBaSnEptzSWTeHE/preview)
+
+* Mainline:
+  * PASSED: **685** (+23)
+  * FAILED: **5**
+  * PASSED [%]: **99.28** (+0,03%)
+
+### Key Changes in testing
+
+1. Added new set of automatic tests for apu6a test setup.
```

Comment:
@StanislawAniola if anything, we enabled more regression tests on apu6a, we did not add them.

Reply:
@miczyg1 this change has not been added by me. As I can see in commits @artur-rs is an author of this sentence. I can change it to something like "Enabled new set of automatic tests for apu6a test setup."

Reply:
Already changed it, because I had something to add too